src/windows/game.ts

```diff
@@ -160,11 +160,12 @@ export function newGameWindow(id: number) {
     window.show()
   })
 
+
   // 设置 GameWindow 的 userAgent
-  window.webContents.userAgent = Global.HttpGetUserAgent
+  window.webContents.userAgent = UserConfigs.window.userAgent
 
   // 载入本地启动器
-  window.loadURL('file://' + path.join(__dirname, '../bin/main/index.html'))
+  window.loadURL('file://' + path.join(__dirname, '../bin/main/index.html'), {userAgent: UserConfigs.window.userAgent})
```

Comment:
hmm, I think it's not necessary to set `User-Agent` for the window. Configuring `webContents.userAgent` should be enough.